### ISSUE TITLE
Download 100%, actually a file is not successful

### ISSUE THREAD

**Issue by cboby1234** [jaylex32/Riptide#2]
Download 100%, actually a file is not successful
![4c9d83cb-51f7-4710-b2c0-6b9cbf96a938](https://github.com/user-attachments/assets/a983816b-d9eb-4bd5-ba37-ece63bafbaad)

**Comment by cboby1234** [comment]
Added Album Quality feature

**Comment by jaylex32** [comment]
> Download 100%, actually a file is not successful ![4c9d83cb-51f7-4710-b2c0-6b9cbf96a938](https://private-user-images.githubusercontent.com/133365135/380618877-a983816b-d9eb-4bd5-ba37-ece63bafbaad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MzAwNzgsIm5iZiI6MTczMDcyOTc3OCwicGF0aCI6Ii8xMzMzNjUxMzUvMzgwNjE4ODc3LWE5ODM4MTZiLWQ5ZWItNGJkNS1iYTM3LWVjZTYzYmFmYmFhZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNFQxNDE2MThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mY2JlZjY4NTc5NTQ1ZTVmOGIyYWQ1YjkyZDAyMTAzNDZkOWUzNTVmYTQ0ZjUyMzI5NWZiZDY5Yjg2MWZmZDUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.p3bhyhWI6mnDlpaPdliaVov1hI0ugdFwSVtWGxM_aMU)

yeah, that sometimes can happen with the device methods, you can first try to restart the app by closing it and reopen it, if that doesn't work go to the settings and try a different device and close the app and reopen and try again.

**Comment by cboby1234** [comment]
Metadata expected to increase genre, several tidal downloads are not displayed

**Comment by jaylex32** [comment]
> Metadata expected to increase genre, several tidal downloads are not displayed

genre metadata is not available in tidal api, there for is impossible to added, new release 1.5.0 have a tons of new features, test it and let me know.

**Comment by lilstavo** [comment]
> Download 100%, actually a file is not successful ![4c9d83cb-51f7-4710-b2c0-6b9cbf96a938](https://github.com/user-attachments/assets/a983816b-d9eb-4bd5-ba37-ece63bafbaad)

Any solution for this?
